### PR TITLE
[BI-37] (re)Installed I18n

### PR DESCRIPTION
**Pull request by alexstefan-binarcode**
Had to reinstall the plugin with the Vue CLI to get rid of some warnings.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,13 +5,15 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint"
+    "lint": "vue-cli-service lint",
+    "i18n:report": "vue-cli-service i18n:report --src \"./src/**/*.?(js|vue)\" --locales \"./src/locales/**/*.json\""
```
Comment (PascuMihai) [inline package.json:9]: we don't need this script, can be removed

File: src/i18n.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import Vue from 'vue'
```
Comment (PascuMihai) [inline src/i18n.js:1]: This should be enough, but test it the translation still works `
import VueI18n from 'vue-i18n'
import Vue from 'vue'
import enLocaleFile from '@/assets/lang/en-US.json'

export default function ({ app }) {
  Vue.use(VueI18n)
  app.i18n = new VueI18n({
    messages: {
      en: enLocaleFile
    },
    locale: 'en',
    fallbackLocale: 'en',
    silentTranslationWarn: true
  })
}`

File: vue.config.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+module.exports = {
```
Comment (PascuMihai) [inline vue.config.js:1]: I don't think this file will be still needed

File: src/i18n.js (hunk 2 above)
Comment (alexstefan-binarcode) [reply src/i18n.js]: Screenshot 2021-08-17 at 12 01 30

**Review by PascuMihai — APPROVED** [review]

**Review by Ceciliaa — APPROVED** [review]

**Pull request merged**